viewing_party/party.py

```diff
@@ -0,0 +1,201 @@
+def create_movie(movie_title, genre, rating):
+    '''
+    function builds a dictionary with three key value 
+    pairs where each value is the relevant parameter for each movie
+    '''
+    if movie_title == None or genre == None or rating == None:
+        return None
+    new_movie = {
+        "title": movie_title,
+        "genre": genre,
+        "rating": rating
+        }
+    print(new_movie)
+    return new_movie
+
+
+def add_to_watched(user_data, movie):
+    '''
+    function adds a specific movie title to the users 
+    list of watched movies
+    '''
+    user_data["watched"].append(movie)
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    '''
+    function adds a specific movie title to the users 
+    list of movies to wacth
+    '''
+    user_data["watchlist"].append(movie)
+    return user_data
+
+
+def watch_movie(user_data, title):
+    '''
+    function checks if movie is in users watchlist. if so, adds 
+    movie to users watched list and removes it from users watchlist. 
+    returns updated user lists
+    '''
+    movies_to_remove = []
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            movies_to_remove.append(movie)
+
+    if len(movies_to_remove) == 0:
+        return user_data
+    else:
+        for movie in movies_to_remove:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
+            
+    return user_data
```

Comment:
I think we can do this all in one for loop! since we are checking if the movie is in watchlist to then add it to watched and remove from watchlist, let's take out the middleman `movies_to_remove` and do something like this:

```suggestion
    movies_to_remove = []
    for movie in user_data["watchlist"]:
        if movie["title"] == title:
            user_data["watchlist"].remove(movie)
            user_data["watched"].append(movie)
            
    return user_data
```